public/index.html

```diff
@@ -9,7 +9,7 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div
       id="root"
-      data-graphql="http://localhost:3333/v1/graphql"
+      data-graphql="http://localhost:3000/v1/graphql"
```

Comment:
@risenW I believe you have React app running on port 3000 so I wouldn't expect to have GraphQL endpoint at the port.

Reply:
But it's ok to leave it as this.

Reply:
I have react running in 3001 locally though